client-misk/src/main/kotlin/app/cash/backfila/client/internal/BackfilaClient.kt

```diff
@@ -1,11 +1,11 @@
-package app.cash.backfila.client.misk.internal
+package app.cash.backfila.client.internal
```

Comment:
it looks like this file is still in the `client-misk` module. Should it be in `client`?

Reply:
still very much WIP. @mpawliszyn and I worked on this for like 40 minutes this afternoon and didn't get too far 😛